subsys/bluetooth/mesh/adv_ext.c

```diff
@@ -65,11 +65,14 @@ static void send_pending_adv(struct k_work *work);
 static bool schedule_send(struct bt_mesh_ext_adv *adv);
 
 static STRUCT_SECTION_ITERABLE(bt_mesh_ext_adv, adv_main) = {
-	.tag = (BT_MESH_LOCAL_ADV |
+	.tag = (
 #if !defined(CONFIG_BT_MESH_ADV_EXT_GATT_SEPARATE)
 		BT_MESH_PROXY_ADV |
 #endif /* !CONFIG_BT_MESH_ADV_EXT_GATT_SEPARATE */
-		BT_MESH_RELAY_ADV),
+#if defined(CONFIG_BT_MESH_ADV_EXT_RELAY_USING_MAIN_ADV_SET)
+		BT_MESH_RELAY_ADV |
```

Comment:
One question. If `CONFIG_BT_MESH_ADV_EXT_RELAY_USING_MAIN_ADV_SET` is not enabled, and `CONFIG_BT_MESH_RELAY_ADV_SETS` is zero, how relay messages are sent?

Reply:
Are you planning to address this?

Reply:
https://github.com/zephyrproject-rtos/zephyr/blob/main/subsys/bluetooth/mesh/adv.c#L162

Reply:
If CONFIG_BT_MESH_ADV_EXT_RELAY_USING_MAIN_ADV_SET not enable and CONFIG_BT_MESH_RELAY_ADV_SETS  is zero, only one queue used, so there are no problem.